utils/dependency_management.py

```diff
@@ -30,7 +30,7 @@
     'graphviz': 'python-graphviz',
 }
 
-CONDA_IGNORE = ['pyblake2', r'.*python_version == \"3\.5\"']
+CONDA_IGNORE = ['pyblake2']
```

Comment:
Not sure what `CONDA_IGNORE` does, but since `pyblake2` is no longer in the dependencies maybe it's also not needed here?